src/App.js

```diff
@@ -1,12 +1,21 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Header from './header/Header';
+import Missions from './components/missions/missions';
+import Rockets from './components/rockets/rocket';
+import Profile from './components/profile/index';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <Header />
+    <Routes>
+      <Route path="/" element={<Rockets />} />
+      <Route path="/rockets" element={<Rockets />} />
+      <Route path="/missions" element={<Missions />} />
+      <Route path="/profile" element={<Profile />} />
+    </Routes>
+  </div>
+);
 
```

Comment:
- [ ] Please I am unable to run your project, please checkout the screenhshot below and kindly fix the issue and ensure the project is running according to the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md). 

![kayongError](https://user-images.githubusercontent.com/65852621/188268368-e3025540-dc51-4218-99cd-ef2b5196d867.png)